weakmap.c

```diff
@@ -274,7 +274,15 @@ wmap_each_i(VALUE key, VALUE val, st_data_t _)
     rb_yield_values(2, key, val);
 }
 
-/* Iterates over keys and objects in a weakly referenced object */
+/*
+ * call-seq:
+ *   map.each {|key, val| ... } -> self
+ *   map.each_pair {|key, val| ... } -> self
```

Comment:
Remove the `each_pair` line.  The documentation guide states "Omit aliases from the call-seq".

Reply:
Yeah, as I said, _now_ I remember it was the old RDoc behavior I relied upon, and we even discussed this earlier :shrug: